transit_odp/organisation/tests/csv/test_service_codes.py

```diff
@@ -320,9 +335,18 @@ def test_csv_output():
         '"Scope Status"',
         '"Seasonal Status"',
         '"Staleness Status"',
-        '"Licence Number"',
-        '"Service Code"',
-        '"Line"',
+        '"Dataset Licence Number"',
```

Comment:
will need update when you update service_codes.py fields

Reply:
Done